shipyard_blueprints/smartsheets/shipyard_smartsheets/cli/download.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import os
+import smartsheet
+import sys
+import logging
+import requests
+import pandas as pd
+from shipyard_templates import ExitCodeException, Spreadsheets as ss
+from typing import Dict, List, Any
+
+# custom exit code
+EXIT_CODE_INVALID_SHEET_ID = 220
```

Comment:
why not in the template ? may have to rename it to something more generic like `EXIT_CODE_INVALID_ID` or `EXIT_CODE_INVALID_INPUT`